server-sdk/src/main/java/com/amazonaws/ml/mms/servingsdk/http/Request.java

```diff
@@ -0,0 +1,17 @@
+package com.amazonaws.ml.mms.servingsdk.http;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Map;
+
+public interface Request {
+    List<String> getHeaderNames();
+    String getRequestURI();
+    String getRequestPath();
+    Map<String, List<String>> getParameterMap();
+    List<String> getParameter(String k);
+    String getContentType();
+    String method();
+    InputStream content() throws IOException;
```

Comment:
the same here, content() is netty way of a getter.

Reply:
True, missed this. I will change this to getInputStream like in servlets